vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -34,6 +36,17 @@ def _get_component_actions(component) -> List[Action]:
     )
 
 
+def _validate_selector_value_NONE(value: Union[SingleValueType, MultiValueType]) -> ValidatedNoneValueType:
+    validated_value: ValidatedNoneValueType = value
+    if value == [NONE_OPTION]:
+        validated_value = [None]
+    elif value == NONE_OPTION:
+        validated_value = None
+    elif isinstance(value, list) and len(value) > 1 and NONE_OPTION in value:
+        validated_value = [i for i in value if i != NONE_OPTION]  # type: ignore[assignment]
```

Comment:
I don't understand why the NONE_OPTION is removed from the list of values in the statement:
```
elif isinstance(value, list) and len(value) > 1 and NONE_OPTION in value:
        validated_value = [i for i in value if i != NONE_OPTION]
```

Isn't that only NONE_OPTION should be replaced with `None` value in a list, as:
```
elif isinstance(value, list) and len(value) > 1 and NONE_OPTION in value:
        validated_value = [i for i in value if i != NONE_OPTION else None]
```

I see it's already covered by unit tests as well, but still don't understand the reason.

Reply:
The reason is that `NONE` is not a valid value, so it would either need to be removed, or replaced.

It is replaced in the cases where it is the single option, but removed in the case where it is among others, as I believe this would be the intuitive case: None + smth = smth.

Do you think we would want to make it so that users can actively use ["foo", "bar",None] or should that not represent ["foo", "bar"]?

Reply:
Oke, I see. We handle "NONE" in the opposite way we handle "ALL" option which absolutely makes sense to me. 

(more like a reminder and heads up then a real question 😄) ⬇️ 
I'm just wondering is it possible to send something like `[1, 2, None, 4]` from Parameter (e.g. vm.Dropdown) to `chart.x` target or `['blue', 'red', None, 'purple']` from another Parameter to `chart.mark.color` target? (probably we encounter similar problem with the "ALL" option as well).

And if it's impossible, are we aware of that limitation?

Reply:
As per above, I think if I understand correctly what you mean, it will not be possible.